Redesign of topic and unit plan webpages

JackMorganNZ opened the pull request:
## Changes

- Simplify topic page by hiding lessons of unit plan (to reinforce that lessons are within unit plans)
- Display unit plan description on sub-page from unit plan page to not hide lessons at end of page. A truncated version of description is provided on the unit plan page.
- Use two column design for topic and unit plan pages on larger screen devices.
- Soften hover state of `link-item` by changing background colour from red to blue.
- Remove underline from headings to clean up pages.
- Add helper function for calculating lesson range for topic and unit plan objects.
- Tidy start of unit plan documents (remove level 2 heading directly after level 1 heading).
- Fix bug where lesson duration wasn't shown.
- Fix bug where language dropdown was aligned in opposite direction.
- Greatly simplify `render_html_field` tag.

Fixes #696 
Fixes #697

## Reviewers

- @saranglove: For design changes
- @hayleyavw / @jordangriffiths01: For technical changes

hayleyavw commented:
It's looking so much better! 😍 
Technical side of things look LGTM, I did notice one design thing when taking a peak around the site,
when viewing a unit plan on a "large screen" I immediately noticed that the space between the main heading and the one below feels a bit close.
<img width="1272" alt="screen shot 2017-12-14 at 6 14 38 pm" src="https://user-images.githubusercontent.com/6937219/33987129-c1a881a4-e0fa-11e7-9897-d1fa6f3ff09c.png">

Also not a graphic designer though....

JackMorganNZ commented:
Ready for review again @hayleyavw @saranglove. I added a bit of padding between the headers (we don't often have two headings against each other hence why it looked tight compared to other pages), and I updated the tests to all use kwargs (must have been args originally and copy/pasting had pulled it through).
